django_select2/__init__.py

```diff
@@ -7,4 +7,7 @@
 .. _Select2: https://select2.org/
 
 """
-default_app_config = "django_select2.apps.Select2AppConfig"
+from django import get_version
+
+if get_version() < '3.2':
```

Comment:
Correct me if I am wrong, but I believe the `<` operator doesn't work on strings. However, it does work as expected on tuples.
```suggestion
if django.VERSION < (3, 2):
```

Reply:
`<` does work with strings (try it in your REPL!), but either approach is fine here.

Reply:
Dang, cool. I learned something new :)